source/common/umf_helpers.hpp

```diff
@@ -279,6 +280,39 @@ inline ur_result_t umf2urResult(umf_result_t umfResult) {
   };
 }
 
+inline umf_result_t createUmfCUDAprovider(
```

Comment:
having `provider` be lower-case stands out a bit. I realise having `CUDAProvider` is also a little awkward.

Reply:
Done.

Reply:
Thanks!